tools/bootloader/usb_flash.c

```diff
@@ -159,12 +159,11 @@ static bool should_restore(void) {
 }
 
 static bool isUpdateRequired(int signed_firmware) {
+  (void)signed_firmware;
```

Comment:
What's going on here?

Reply:
An unused variable will generate a compiler warning. Compiler warnings are treated as errors in our build. (void) is the technique used to keep an unused var in the code but prevent the warning. This is some legacy stuff that someone thought might be useful at some point though we aren't currently doing anything with it.